### PR TITLE
fixed TypeError

### PR DESCRIPTION
**Pull request by raghuramshankar**
#### Reference issue

#409 
#### What does this implement/fix?

Fixed the bug where when you click outside the figure, it throws TypeError. Now it just ignores the outside clicks.

### CODE REVIEW

File: ArmNavigation/two_joint_arm_to_point_control/two_joint_arm_to_point_control.py [hunk 1]
```diff
@@ -54,11 +59,17 @@ def two_joint_arm(GOAL_TH=0.0, theta1=0.0, theta2=0.0):
...
+            y = y_prev
 
         wrist = plot_arm(theta1, theta2, x, y)
 
         # check goal
-        d2goal = np.hypot(wrist[0] - x, wrist[1] - y)
+        try:
+            d2goal = np.hypot(wrist[0] - x, wrist[1] - y)
+        except TypeError:
+            pass
```
Comment (AtsushiSakai) [inline ArmNavigation/two_joint_arm_to_point_control/two_joint_arm_to_point_control.py:72]: This empty except is needed?

**Review by AtsushiSakai — CHANGES_REQUESTED** [review]
Thank you for your PR again. I have a small comment. PTAL.

File: ArmNavigation/two_joint_arm_to_point_control/two_joint_arm_to_point_control.py (hunk 1 above)
Comment (raghuramshankar) [reply ArmNavigation/two_joint_arm_to_point_control/two_joint_arm_to_point_control.py:72]: Hi, thank you for your review. Yes, if this error is not passed, the same bug appears again as the value of x and y are "NoneType". I feel it is needed,

Comment (AtsushiSakai) [reply ArmNavigation/two_joint_arm_to_point_control/two_joint_arm_to_point_control.py:72]: OK. So could you please use if statement instead of try catch.

**Review by AtsushiSakai — APPROVED** [review]
LGTM. Thank you!!

**Pull request merged**